lib/src/inline_parser.dart

```diff
@@ -871,9 +875,19 @@ class DelimiterRun implements Delimiter {
       'canClose: $canClose>';
 }
 
+class DelimiterTag {
+  DelimiterTag(this.tag, this.indicators);
+
+  // Tag name of AST Element
+  final String tag;
+
+  /// The length of [indicators]
+  final int indicators;
```

Comment:
The name of this field is probably not the best (as seen by the doc comment above it).

I would expect a field named 'indicators' to the String perhaps, or a List. This should be renamed to something like... 'indicatorLength'. If that is too long, I'd be open to other ideas...

Reply:
I have changed it to indicatorLength. Yes, indicatorLength is too long, this was the reason why I named it indicators.